BackEnd/README.md

```diff
@@ -0,0 +1,15 @@
+# PunORama
+
+I'm using the Linux native English dictionary (/usr/share/dict/american-english), to find puns.
+
+In order to obtain words that are similar to a user's input, I used a [word2vec](https://code.google.com/archive/p/word2vec/) pre-trained word vector. Google published pre-trained vectors trained on part of Google News dataset (about 100 billion words). The model contains 300-dimensional vectors for 3 million words and phrases.
+
+I ran the `distance` binary on my entire dictionary to get the 10 semantically closest words to each word of the dictionary.
+
+`/output/file > distance < cat /usr/share/dict/american-english`
```

Comment:
Computer code should be spaced 4 spaces from the left column; this tells Markdown what it is.